pkg/sosreport/mcp/commands.go

```diff
@@ -0,0 +1,142 @@
+package sosreport
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"regexp"
+	"github.com/ovn-kubernetes/ovn-kubernetes-mcp/pkg/sosreport/types"
+)
+
+const defaultMaxLines = 100
+
+// getCommandOutput reads a command output file by filepath from manifest
+func getCommandOutput(sosreportPath, relativeFilepath, pattern string, maxLines int) (string, error) {
+	if err := validateSosreportPath(sosreportPath); err != nil {
+		return "", err
+	}
+
+	fullPath := filepath.Join(sosreportPath, relativeFilepath)
+	if _, err := os.Stat(fullPath); os.IsNotExist(err) {
+		return "", fmt.Errorf("command output file not found: %s", relativeFilepath)
+	}
+
+	file, err := os.Open(fullPath)
+	if err != nil {
+		return "", fmt.Errorf("failed to open file: %w", err)
+	}
+	defer file.Close()
+
+	var searchPattern *regexp.Regexp
+	if pattern != "" {
+		searchPattern, err = regexp.Compile(pattern)
+		if err != nil {
+			return "", fmt.Errorf("invalid pattern: %w", err)
+		}
+	}
+
+	if maxLines == 0 {
+		maxLines = defaultMaxLines
+	}
+
+	output, err := readWithLimit(file, searchPattern, maxLines)
+	if err != nil {
+		return "", err
+	}
+
+	if output == "" && pattern != "" {
+		return fmt.Sprintf("No lines matching pattern %q found\n", pattern), nil
+	}
+
+	return output, nil
+}
+
+// listPlugins returns a list of enabled plugins with their command counts
+func listPlugins(sosreportPath string) (types.ListPluginsResult, error) {
```

Comment:
nice plugin aggregation for the collected logs from sos report's manifest.json file !
what about the remaining logs collected outside of `sos_commands` directory. 
For example need to analyze ovs-vswitchd and ovsdb-server.log logs from `/var/log/openvswitch` directory.